Apps/WebClient/src/NewClientApp/src/components/private/reports/NotesReportComponent.vue

```diff
@@ -57,34 +56,27 @@ const fields: ReportField[] = [
 const logger = container.get<ILogger>(SERVICE_IDENTIFIER.Logger);
 const noteStore = useNoteStore();
 
-const notesAreLoading = computed<boolean>(() => noteStore.notesAreLoading);
-
-const notes = computed<UserNote[]>(() => noteStore.notes);
-
-const visibleRecords = computed<UserNote[]>(() => {
-    const records = notes.value.filter((record) =>
-        props.filter.allowsDate(record.journalDate)
-    );
-    records.sort((a, b) => {
-        const firstDate = new DateWrapper(a.journalDate);
-        const secondDate = new DateWrapper(b.journalDate);
-
-        if (firstDate.isBefore(secondDate)) {
-            return 1;
-        }
-
-        if (firstDate.isAfter(secondDate)) {
-            return -1;
-        }
-
-        return 0;
-    });
-    return records;
-});
-
-const isEmpty = computed<boolean>(() => visibleRecords.value.length === 0);
-
-const items = computed<UserNoteRow[]>(() => {
+const notesAreLoading = computed(() => noteStore.notesAreLoading);
+const visibleRecords = computed(() =>
+    noteStore.notes
+        .filter((record) => props.filter.allowsDate(record.journalDate))
+        .sort((a, b) => {
+            const firstDate = new DateWrapper(a.journalDate);
+            const secondDate = new DateWrapper(b.journalDate);
+
+            if (firstDate.isBefore(secondDate)) {
+                return 1;
+            }
+
+            if (firstDate.isAfter(secondDate)) {
+                return -1;
+            }
+
+            return 0;
+        })
```

Comment:
This is about the third time this dateWrapper sorting function is being defined, let's merge this into the current PR and I'll work to create a sorting utility to manage the different sorting methods employed, I believe there are a few in the stores that could be extracted into a encapsulated utility.